terraform/modules/galileo-eks/variables.tf

```diff
@@ -13,7 +13,7 @@ variable "cluster_name" {
 variable "cluster_version" {
   description = "Kubernetes `<major>.<minor>` version to use for the EKS cluster (i.e.: `1.22`)"
   type        = string
-  default     = "1.24"
+  default     = "1.28"
```

Comment:
There was some constraint on 1+ of our services on why we couldn't use more recent versions of Kubernetes, which is why we had this on an older version. Would be great to validate that it works.

Reply:
Interesting, will do!  I know support is ending in a couple weeks, and more expensive extended support didn't sound appealing :P

Reply:
Agreed, we've been lagging behind on actually updating them reliably.